rasa/core/training/interactive.py

```diff
@@ -75,6 +75,8 @@
 
 MAX_VISUAL_HISTORY = 3
 
+
```

Comment:
This is made according to style sheet?

Reply:
I don't remember doing that. I"ll change it.